playwright/lib/util.ts

```diff
@@ -27,41 +29,51 @@ const getHost = (stage?: Stage | undefined) => {
 	return hostnames[stage ?? getStage()] ?? hostnames.dev;
 };
 
+const getDcrContentType = (
+	type: ContentType,
+): 'Article' | 'Front' | 'TagFront' => {
+	switch (type) {
+		case 'front':
+			return 'Front';
+
+		case 'taggedFront':
+			return 'TagFront';
```

Comment:
Minor, but if the DCR type is TagFront should we be using _tagFront_ instead of _taggedFront_ in the ContentType type for consistency?